project/YaoKuang/README.rst

```diff
@@ -0,0 +1,87 @@
+SimpleArray for Multi-Dimensional Field
+
+(Modmesh) SimpleArray: a simple C++ library for multi-dimensional arrays that provide seamless interface with numpy ndarray. Download this file.
+
+Basic Information
+
+SimpleArray is part of modmesh, and the code is house in its repository: https://github.com/solvcon/modmesh
```

Comment:
I strongly suggest you to create your own repository for code development.  It takes significant efforts to get familiar with modmesh development flow.